go/adbc/driver/flightsql/utils.go

```diff
@@ -32,7 +32,7 @@ func adbcFromFlightStatus(err error) error {
 	switch status.Code(err) {
 	case codes.OK:
 		return nil
-	case codes.Canceled:
+	case codes.Canceled, codes.DeadlineExceeded:
```

Comment:
there's a separate TIMEOUT status: https://github.com/apache/arrow-adbc/blob/fc01c31c95b48cdc22f0780d1498d89dd20563f9/go/adbc/adbc.go#L134

Reply:
I missed that, i'll fix that.